test/tools/IsolTestOptions.cpp

```diff
@@ -76,6 +76,8 @@ bool IsolTestOptions::parse(int _argc, char const* const* _argv)
 		return false;
 	}
 	enforceViaYul = true;
+	enforceGasTest = (evmVersion() == langutil::EVMVersion::istanbul());
```

Comment:
Again, `EVMVersion{}`:
```suggestion
	enforceGasTest = (evmVersion() == langutil::EVMVersion{});
```